Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsCardPaddingDecorator.kt

```diff
@@ -22,15 +22,17 @@ class StatisticsCardPaddingDecorator(
         val resources = parent.context.resources
 
         val adapter = parent.adapter
+        val distance = resources.getDimensionPixelSize(cardDistance)
```

Comment:
The card padding is larger than on the mockups now though. On the mockups the horizontal distance between cards is half of the distance between vertical cards. I think both look okay, and we can merge this because looks better than if it's inconsistent.

What do you think? @chiljamgossow

Reply:
I corrected it